http/org.wso2.carbon.transport.http.netty/src/test/java/org/wso2/carbon/transport/http/netty/util/clients/http2/HTTP2ClientInitializer.java

```diff
@@ -15,7 +15,7 @@
 * specific language governing permissions and limitations
 * under the License.
 */
-package org.wso2.carbon.transport.http.netty.util.client.http2;
+package org.wso2.carbon.transport.http.netty.util.clients.http2;
```

Comment:
Any particular reason for the package rename ? As a best practice, we don't use plural in package names